### PR TITLE
Docs: Control Port not always enabled by default

### PR DESCRIPTION
**Pull request by JBaczuk**
added note that for tor to work in automatic mode, the control port must be enabled and also added how to do that by using torrc config file.

### CODE REVIEW

**Comment by fanquake** [comment]
Please remove the trailing whitespace from the lines you've added. That is what's causing the tests to fail.

File: doc/tor.md [hunk 1]
```diff
@@ -102,8 +102,9 @@ and, if not disabled, configured using the `-torcontrol` and `-torpassword` sett
 To show verbose debugging information, pass `-debug=tor`.
 
 Connecting to Tor's control socket API requires one of two authentication methods to be 
-configured. For cookie authentication the user running bitcoind must have write access 
-to the `CookieAuthFile` specified in Tor configuration. In some cases this is 
+configured. It also requires the control socket to be enabled, eg. put `ControlPort 9051`
+in torrc config file. For cookie authentication the user running bitcoind must have write
```
Comment (practicalswift) [inline doc/tor.md]: Nit: Use \`torrc\`

File: doc/tor.md [hunk 2]
```diff
@@ -102,8 +102,9 @@ and, if not disabled, configured using the `-torcontrol` and `-torpassword` sett
 To show verbose debugging information, pass `-debug=tor`.
 
 Connecting to Tor's control socket API requires one of two authentication methods to be 
-configured. For cookie authentication the user running bitcoind must have write access 
-to the `CookieAuthFile` specified in Tor configuration. In some cases this is 
+configured. It also requires the control socket to be enabled, eg. put `ControlPort 9051`
```
Comment (practicalswift) [inline doc/tor.md]: Nit: "e.g."

File: doc/tor.md [hunk 3]
```diff
@@ -93,23 +93,24 @@ API, to create and destroy 'ephemeral' hidden services programmatically.
...
-preconfigured and the creation of a hidden service is automatic. If permission problems 
-are seen with `-debug=tor` they can be resolved by adding both the user running tor and 
-the user running bitcoind to the same group and setting permissions appropriately. On 
-Debian-based systems the user running bitcoind can be added to the debian-tor group, 
-which has the appropriate permissions. An alternative authentication method is the use 
-of the `-torpassword` flag and a `hash-password` which can be enabled and specified in 
+Connecting to Tor's control socket API requires one of two authentication methods to be
+configured. It also requires the control socket to be enabled, e.g. put `ControlPort 9051`
+in `torrc` config file. For cookie authentication the user running bitcoind must have write
+access to the `CookieAuthFile` specified in Tor configuration. In some cases this is
```
Comment (laanwj) [inline doc/tor.md]: This is not your doing, but I think this statement is not correct. It needs read access to the cookie file, not write access.

Comment (kewde) [reply doc/tor.md]: Seems like the author copied that wording from the tor spec. The user running the tor daemon needs the write access, and the user running bitcoind should only need read.

Comment (laanwj) [reply doc/tor.md]: exactly

**Comment by laanwj** [comment]
Closing this, let me know if you want to pick it up again, but this is taking much too long for a documentation-only change

**Comment by JBaczuk** [comment]
Apologies, what exactly is holding this up?

**Comment by MarcoFalke** [comment]
@JBaczuk Could you please address the in-line feedback?

File: doc/tor.md (hunk 3 above)
Comment (JBaczuk) [reply doc/tor.md]: ok I've made this change, apologies, I didn't realize this was holding this PR up.

**Comment by JBaczuk** [comment]
@MarcoFalke oh, ok done.

**Comment by MarcoFalke** [comment]
This pull is still closed and it won't let me reopen due to the push.

**Comment by JBaczuk** [comment]
I didn't know that could happen, what's the proper procedure here?

**Comment by MarcoFalke** [comment]
There hasn't been any substantial discussion, so feel free to open a new pull request in this case.

**Comment by JBaczuk** [comment]
Ok reopened as  #14120

**Pull request closed without merging**